ospf6d/ospf6_asbr.h

```diff
@@ -58,6 +58,7 @@ struct ospf6_as_external_lsa {
 	/* followed by none or one external route tag */
 	/* followed by none or one referenced LS-ID */
 };
+struct ospf6 *o;
```

Comment:
This is not a good idea, after rebasing with `master` this causes compilation error: duplicated declarations.